src/action/main.ts

```diff
@@ -112,7 +112,7 @@ async function gitHubAppToken(app: GitHubAppInfo | undefined, gitHubApiUrl: stri
 
   const response = type === 'app'
     ? await auth({type: 'app'})
-    : (app.installation ? await auth({type: 'installation', installationId: app.installation.value}) : undefined)
+    : (app.installation ? await auth({type: 'installation', installationId: app.installation.value, refresh: true}) : undefined)
```

Comment:
this is from their docs at https://github.com/octokit/auth-app.js?tab=readme-ov-file#installation-authentication

<img width="990" alt="image" src="https://github.com/user-attachments/assets/a1e43e1a-5427-4e36-ba09-336994677ee9">